README.md

```diff
@@ -16,9 +16,9 @@ Search the log and corresponding diff at once. Notice that when you use `|` the
 
 ## Installing
 
-`fzf` is **required**:
 ```bash
-brew install fzf
+brew tap AppleBoiy/git-fuzzy
```

Comment:
I'm not OK hosting my program on someone else's tap. It's not that your tap is specifically a problem, but it's a general trust thing.

I would not trust an app that is developed by one user, but self-hosted in brew by another user.